tools/driver/swift_symbolgraph_extract_main.cpp

```diff
@@ -260,16 +260,16 @@ int swift_symbolgraph_extract_main(ArrayRef<const char *> Args, const char *Argv
   // don't need to print these errors.
   CI.removeDiagnosticConsumer(&DiagPrinter);
   
-  for (const auto &ModuleName : VisibleModuleNames) {
```

Comment:
Do we still need to accumulate the `VisibleModuleNames` above?  Maybe we should log what `findDeclaredCrossImportOverlaysTransitive` returns?

Reply:
As for logging, since we're printing the modules as we load them, i don't think it's 100% necessary to add more logging on top of that just to list the modules beforehand.